File: auto-docs/executables/julia/basic_bar.jl
# Learn about API authentication here: {{BASE_URL}}/julia/getting-started
# Find your api_key here: {{BASE_URL}}/settings/api

using Plotly

Plotly.signin("TestBot", "r1neazxo9w")
data = [
  [
    "x" => ["giraffes", "orangutans", "monkeys"], 
    "y" => [20, 14, 23], 
    "type" => "bar"
  ]
]
response = Plotly.plot(data, ["filename" => "basic-bar", "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/matlab/basic_bar.m
signin('TestBot', 'r1neazxo9w')
data = {...
  struct(...
    'x', { {'giraffes', 'orangutans', 'monkeys'} }, ...
    'y', [20, 14, 23], ...
    'type', 'bar')...
};
response = plotly(data, struct('auto_open', false, 'fileopt', 'overwrite', 'filename', 'basic-bar'));
plot_url = response.url